influxdb3_cache/src/meta_cache/mod.rs

```diff
@@ -601,88 +167,12 @@ mod tests {
-                    "+---------+------+",
-                    "| region  | host |",
-                    "+---------+------+",
-                    "| us-east | a    |",
-                    "| us-east | b    |",
-                    "+---------+------+",
-                ],
-            },
-            TestCase {
-                desc: "eq predicate on host",
-                predicates: &[Predicate::new_eq(host_col_id, "h")],
-                expected: &[
-                    "+---------+------+",
-                    "| region  | host |",
-                    "+---------+------+",
-                    "| ca-cent | h    |",
-                    "+---------+------+",
-                ],
-            },
-            TestCase {
-                desc: "eq predicate on region and host",
-                predicates: &[
-                    Predicate::new_eq(region_col_id, "ca-cent"),
-                    Predicate::new_eq(host_col_id, "h"),
-                ],
-                expected: &[
-                    "+---------+------+",
-                    "| region  | host |",
-                    "+---------+------+",
-                    "| ca-cent | h    |",
-                    "+---------+------+",
-                ],
-            },
-            TestCase {
-                desc: "not eq predicate on region",
-                predicates: &[Predicate::new_not_eq(region_col_id, "ca-cent")],
-                expected: &[
-                    "+---------+------+",
-                    "| region  | host |",
-                    "+---------+------+",
-                    "| ca-east | e    |",
-                    "| ca-east | f    |",
-                    "| eu-cent | k    |",
-                    "| eu-cent | l    |",
-                    "| eu-east | i    |",
-                    "| eu-east | j    |",
-                    "| us-east | a    |",
-                    "| us-east | b    |",
-                    "| us-west | c    |",
-                    "| us-west | d    |",
-                    "+---------+------+",
-                ],
-            },
-            TestCase {
-                desc: "not eq predicate on region and host",
-                predicates: &[
-                    Predicate::new_not_eq(region_col_id, "ca-cent"),
-                    Predicate::new_not_eq(host_col_id, "a"),
-                ],
-                expected: &[
-                    "+---------+------+",
-                    "| region  | host |",
-                    "+---------+------+",
-                    "| ca-east | e    |",
-                    "| ca-east | f    |",
-                    "| eu-cent | k    |",
-                    "| eu-cent | l    |",
-                    "| eu-east | i    |",
-                    "| eu-east | j    |",
-                    "| us-east | b    |",
-                    "| us-west | c    |",
-                    "| us-west | d    |",
-                    "+---------+------+",
-                ],
-            },
```

Comment:
All of these test cases were removed because the `Eq` and `NotEq` variants were removed from the `Predicate` type.